plugins/item_tasks/web_client/views/SelectSingleFileTaskWidget.js

```diff
@@ -1,15 +1,15 @@
 import View from 'girder/views/View';
-
 import 'girder/utilities/jquery/girderEnable';
 import 'girder/utilities/jquery/girderModal';
 
 import PaginateTasksWidget from './PaginateTasksWidget';
-import ItemTaskCollection from '../collections/ItemTaskCollection';
 
-import SelectTaskViewDialogTemplate from '../templates/selectTaskViewDialog.pug';
-import selectTaskViewDescriptionTemplate from '../templates/selectTaskViewTargetDescription.pug';
-
-import '../stylesheets/selectTaskView.styl';
+import ItemTaskCollection from '../collections/ItemTaskCollection';
+import SelectSingleFileTaskWidgetTemplate from
+'../templates/selectSingleFileTaskWidget.pug';
```

Comment:
We're more permissive about 100-character wrapping in Javascript files. I would leave these imports all on the same line.